### ISSUE TITLE
Feature Request: add framework to implement custom loss functions (example: nbeats_ensemble)

### ISSUE THREAD

**Issue by lg1000** [business-science/modeltime.gluonts#47]
Hello Matt,

in order to achieve high accuracy, I want to implement my own loss functions. For example, as I have read in the original nbeats paper, there is a custom loss function already in use (OWA). 
However, in the nbeats documentation I am told to use either MAPE or MASE as loss metric. I do not like to rely on those metrics, for numerous reasons.
There are several documentations on how to define a custom loss function in keras. This does not seem very challenging.
Nevertheless, I do not know, how to make the loss function available to R via the reticulate interface.
Is there a way to create a yardstick-like framework, in order to create custom metrics?
